src/Symfony/Bundle/FrameworkBundle/Resources/config/property_info.php

```diff
@@ -45,6 +47,7 @@
             ->tag('property_info.type_extractor', ['priority' => -1002])
             ->tag('property_info.access_extractor', ['priority' => -1000])
             ->tag('property_info.initializable_extractor', ['priority' => -1000])
+            ->tag('property_info.attributes_extractor', ['priority' => -1000])
```

Comment:
```suggestion
            ->tag('property_info.attribute_extractor', ['priority' => -1000])
```
To be consistent? (same in `FrameworkExtension` and `PropertyInfoPass`)